### ISSUE TITLE
Parameter values are ignored if MySqlParameterCollection.Insert is used.

### ISSUE THREAD

**Issue by xtomik** [mysql-net/MySqlConnector#869]
If MySqlParameter is inserted into MySqlCommand.Parameters collection using MySqlParameterCollection.Insert, then the MySqlParameter.Value is not used.
```
var command = connection.CreateCommand();
var parameter = new MySqlParameter("@Id", MySqlDbType.Int32);
command.Parameters.Insert(0, parameter); 
parameter.Value = 5;
command.CommandText = "SELECT @Id";
var value = command.ExecuteScalar(); // returns DBNull
```

**Comment by bgrainger** [comment]
Pre-release fix available in 1.0.1-beta.0.22 at GitHub Packages: https://github.com/mysql-net/MySqlConnector/packages/39735

**Comment by bgrainger** [comment]
Fixed in [1.0.1](https://github.com/mysql-net/MySqlConnector/releases/tag/1.0.1).
